backend/server.js

```diff
@@ -410,17 +411,73 @@ app.get('/banks/:bankId/accounts/:accountId/transactions', async (req, res, next
 });
 
 app.post("/create-payment-intent", async (req, res) => {
-	const { items } = req.body;
+	//const { items } = req.body;
 	// Create a PaymentIntent with the order amount and currency
 	const paymentIntent = await stripe.paymentIntents.create({
-	  amount: 2000,
-	  currency: "usd"
+	    payment_method_types: ['card'],
+		amount: 1000,
+		currency: 'usd',
+		application_fee_amount: 1000*0.029+30,
+		transfer_data: {
+		destination:'acct_1HqWTpQLXrRrQtsS',
+  },
 	});
 	res.send({
 	  clientSecret: paymentIntent.client_secret
 	});
 });
 
+
+app.post("/onboard-user", async (req, res) => {
+	try {
+	  const account = await stripe.accounts.create({type: "express"});
+	  console.log('stripe account id: ' + account.id)
+	  req.session.accountID = account.id;
+	  console.log('req session: '+req.session.accountID);
+	  const addStripeAccount = { $set: {hasStripeAccount: true, stripeAccountId: account.id}}
+	  User.updateOne({username:req.user.username}, addStripeAccount, (err,result)=>{
+		 console.log('in mongo query: '+result);
+	  })
+	  const origin = `${req.headers.origin}`;
+	  const accountLinkURL = await generateAccountLink(account.id, origin);
+	  res.send({url: accountLinkURL});
+	} catch (err) {
+	  res.status(500).send({
+		error: err.message
+	  });
+	}
+  });
+
+  app.get("/onboard-user/refresh", async (req, res) => {

```

Comment:
This function should be on the frontend, not the backend. It doesn't work properly.